juniper_official/Linecard/fi-statistics.rule

```diff
@@ -107,6 +107,7 @@ healthbot {
                     then {
                         status {
                             color green;
+                            message "Center chip fabric in error count:$cchip-fi-error-packets";
```

Comment:
Should we say that this "is normal" as well?

Reply:
Yes, changed